frontend/apps/marketing/src/components/actionBlocks/fullWidthActionBlock/FullWidthActionBlock.tsx

```diff
@@ -0,0 +1,71 @@
+import {BaseEntry, EntryFields} from 'contentful';
+
+import DSCOFullWidthActionBlock, {
+  ActionBlockProps,
+} from '@code-dot-org/component-library/actionBlock/fullWidthActionBlock';
+
+export type FullWidthActionBlockContentfulProps = ActionBlockProps & {
+  image: BaseEntry & {
+    fields: {
+      description?: EntryFields.Text;
+      title?: EntryFields.Text;
+      file: {url: EntryFields.Text};
+    };
+  };
+  overline: EntryFields.Text;
+  title: EntryFields.Text;
+  description: EntryFields.Text;
+  primaryButton: BaseEntry & {
+    fields: {
+      label: EntryFields.Text;
+      primaryTarget: EntryFields.Text;
+      ariaLabel: EntryFields.Text;
+    };
+  };
+  secondaryButton: BaseEntry & {
+    fields: {
+      label: EntryFields.Text;
+      primaryTarget: EntryFields.Text;
+      ariaLabel: EntryFields.Text;
+    };
+  };
+  background: EntryFields.Text;
+};
+
+const FullWidthActionBlock: React.FC<FullWidthActionBlockContentfulProps> = ({
+  image,
+  overline,
+  title,
+  description,
+  primaryButton,
+  secondaryButton,
+  background,
+}) => (
+  <DSCOFullWidthActionBlock
```

Comment:
Could we add some unit tests considering all the optional chaining being done here?

Reply:
Yes! Should I primarily test the buttons or all of the props? 

And I'll add tests to ActionBlock too since it's similar.